.github/workflows/ci.yml

```diff
@@ -27,7 +27,7 @@ jobs:
     strategy: 
       fail-fast: false
       matrix: 
-        rails: [norails, rails60, rails52, rails51, rails42, rails41, rails40, rails32, rails31, rails30]
+        rails: [norails, rails61, rails60, rails52, rails51, rails42, rails41, rails40, rails32, rails31, rails30]
```

Comment:
I added a test environment for Rails 6.1 which actually exposed the existing bug.